### PR TITLE
Update the updater code to check for the newest SnapshotHelper version

### PR DESCRIPTION
**Pull request by i2amsam**
Change the `updater` to look specifically for the comment at the end of the `SnapshotHelper` file

### CODE REVIEW

File: lib/assets/SnapshotHelper.swift [hunk 1]
```diff
@@ -105,4 +105,4 @@ extension XCUIElement {
 
 // Please don't remove the lines below
 // They are used to detect outdated configuration files
-// SnapshotHelperVersion [[1.1]]
+// SnapshotHelperVersion [1.1]
```
Comment (KrauseFx) [inline lib/assets/SnapshotHelper.swift]: Any reason why you switched from 2 to only 1 `[`?

File: lib/snapshot/runner.rb [hunk 2]
```diff
@@ -226,7 +226,7 @@ def verify_helper_is_current
       helper_files.each do |path|
         content = File.read(path)
 
-        if content.include?("start.pressForDuration(0, thenDragToCoordinate: finish)")
+        unless content.include?("SnapshotHelperVersion [1.1]")
```
Comment (KrauseFx) [inline lib/snapshot/runner.rb]: We still need to support the other migration, so this should actually be multiple `if`s in this method

File: lib/assets/SnapshotHelper.swift (hunk 1 above)
Comment (i2amsam) [inline lib/assets/SnapshotHelper.swift]: Seemed like only 1 `[` seemed necessary

File: lib/snapshot/runner.rb (hunk 2 above)
Comment (i2amsam) [inline lib/snapshot/runner.rb]: Wouldn't the `[1.1]` check get that case too? (since the `1.1` version does not have the `start.pressForDuration` bit)

Comment (KrauseFx) [inline lib/snapshot/runner.rb]: Mh, I guess that's true, makes sense :+1:

**Comment by KrauseFx** [comment]
Looks good :rocket:

**Pull request merged**